en/alerts-notifications/resources-status.md

```diff
@@ -194,3 +205,27 @@ The service panel contains several informative tabs:
 
 If an acknowledgement or downtime is set on the service, it will be displayed in
 the panel and the header will be accordingly colored.
+
+#### Graph
+
+The graph tab enables you to visually display how the metrics evolves for the selected resource. Hovering the metric curves will display a tooltip containing the different values for the corresponding point in time.
+
+It’s possible to select or deselect metrics for display using the legend. By clicking on a legend item, it will unselect all metrics other than this one. Clicking on a metric being the only one selected will do the opposite: unselect it and select all others:
```

Comment:
```suggestion
It's possible to select or deselect metrics for display using the legend. By clicking on a legend item, it will unselect all metrics other than this one. Clicking on a metric being the only one selected will do the opposite: unselect it and select all others:
```